File: SourceCode/C/day025.c
/* Author: Ahmed A. M. ALOMAR
Date: January 25 - 2020
Description:
This program is an example for the Logical operators */


#include <stdio.h>

int main()
{
  _Bool x = 10;
  _Bool y = 20;
  _Bool logicalAnd;
  _Bool logicalOr;
  _Bool logicalNot;

  // The AND operator
  logicalAnd = x && y;

  // The OR operator
  logicalOr = (x = 10) || (y = 10);

  // The NOT operator
  logicalNot = !x;

  printf("Logical AND result: %d\n", logicalAnd);
  printf("Logical OR result: %d\n", logicalOr);
  printf("Logical NOT result: %d", logicalNot);

  return 0;
}

File: SourceCode/Python/day025.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 5 - 2020
#
# Description:
# * Example of deleting data from a
#   Dictionary
# ------------------------------------ #

animals = {
  "bird": "eagle",
  "mammal": "tiger",
  "fish": "shark",
  "reptiles": "alegator"
}

# Using pop()
pet = animals.pop("bird")

print(pet)
print(animals)

# using the del keyword
del animals["reptiles"]

print(animals)

# Using clear()
animals.clear()

print(animals)
